docs/fundamentals/executor/containerize-executor.md

```diff
@@ -0,0 +1,168 @@
+# Dockerize your Executor
+
+Once you have understood what an `Executor` is and how it can be used inside a `Flow`, you may be interested in wrapping this Executor into a container
+so that you can isolate its dependencies and make it ready to run in the cloud or in Kubernetes.
+
+One option is to leverage {ref}`Jina Hub <hub/index>` infrastructure to make sure your Executor can run as a container.
+
+However, you can also build a `docker` image yourself and use it like any other Executor. There are some requirements
```

Comment:
```suggestion
However, you can also build a Docker image yourself and use it like any other Executor. There are some requirements
```
We're not talking about the command `docker`, but rather the "thing" Docker. So no backticks